loader-tests/check-opml.ts

```diff
@@ -59,6 +60,10 @@ cli.run(async args => {
   }
 
   let feeds = await parseFeedsFromFile(opmlFile)
+
+  const jobsCount = home ? feeds.length * 2 : feeds.length
```

Comment:
Run `eslint --fix .` to fix code style.

And you marked `Run pnpm test` as done, but didn’t call it ;).

Reply:
fixed 👀

Reply:
I called, but it seems these errors were hidden under a lot of 
```
Unable to read file core/coverage/tmp/coverage-49309-1715592007988-0.json
ENOENT: no such file or directory, open /core/coverage/tmp/coverage-49309-1715592007988-0.json
```

Want look at this errors later